### PR TITLE
http://dotcms/core\#12656 Style the confirmation dialog

### PR DESCRIPTION
**Pull request by luisrojascr**

### CODE REVIEW

File: src/app/portlets/content-types/edit/content-types-edit.component.html [hunk 1]
```diff
@@ -16,9 +16,9 @@
         (removeFields)="removeFields($event)">
     </content-type-fields-drop-zone>
 </content-type-layout>
-<p-confirmDialog width="425" #cd>
+<p-confirmDialog width="400" #cd>
```
Comment (fmontes) [inline src/app/portlets/content-types/edit/content-types-edit.component.html]: Move this code to the ```app.component.html``` and test, this should be global.

File: src/app/portlets/content-types/fields/content-type-fields-row/content-type-fields-row.component.ts [hunk 2]
```diff
@@ -37,16 +45,21 @@ export class ContentTypeFieldsRowComponent extends BaseComponent {
...
-        this.fieldRow.columns = this.fieldRow.columns.map(col => {
-            const index: number = col.fields.indexOf(field);
-
-            if (index !== -1) {
-                col.fields.splice(index, 1);
-             }
-            return col;
+        this.confirmationService.confirm({
+            accept: () => {
+                this.fieldRow.columns = this.fieldRow.columns.map(col => {
```
Comment (fmontes) [inline src/app/portlets/content-types/fields/content-type-fields-row/content-type-fields-row.component.ts]: Don't understand why this code is here... as far as I can see the accept should only emmit isn't? if not... please move this to an external method.

File: src/styles/primeng-theme/components/_dialog.scss [hunk 3]
```diff
@@ -29,11 +30,29 @@
         button {
             margin: 0;
             float: none;
+            &.button-delete{
+                box-shadow: none;
+                span{
+                    color: $blue;
+                }
+                &:hover{
+                    background: $gray-lighter;
```
Comment (fmontes) [inline src/styles/primeng-theme/components/_dialog.scss]: Create var for dialog.

File: src/app/api/services/dot-confirmation-service.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { Injectable, EventEmitter } from '@angular/core';
+import { FooterLabels } from './../../shared/models/dot-confirmation/footer-labels.model';
+import { DotConfirmation } from './../../shared/models/dot-confirmation/dot-confirmation.model';
+import { BehaviorSubject } from 'rxjs';
+import { MessageService } from './messages-service';
+import { ConfirmationService } from 'primeng/primeng';
+
+@Injectable()
+export class DotConfirmationService {
```
Comment (freddyucv) [inline src/app/api/services/dot-confirmation-service.ts]: Doc and testing

File: src/app/portlets/content-types/edit/content-types-edit.component.ts [hunk 5]
```diff
@@ -77,14 +80,20 @@ export class ContentTypesEditComponent extends BaseComponent implements OnInit {
      * @memberof ContentTypesEditComponent
      */
     public deleteContentType($event): void {
-        this.confirmationService.confirm({
+        this.dotConfirmationService.confirm({
```
Comment (freddyucv) [inline src/app/portlets/content-types/edit/content-types-edit.component.ts]: do this method have test?

**Pull request merged**